_layouts/doc.html

```diff
@@ -4,7 +4,7 @@
 <div class="container">
   {% include doc-heading.html %}
   <div class="more-bottom white-box page-content">
-    {% if page.permalink == '/doc/' %}
+    {% if page.permalink == doc_link %}
```

Comment:
I guess this means even `/doc/` itself will be translated?

Does this mean we have to make similar replacements everywhere? For example, I think I recently used `_doc/` somewhere.

Reply:
> I guess this means even `/doc/` itself will be translated?

Yes, you can see example at https://wwwtest.qubes-os.org/de/doc/

> Does this mean we have to make similar replacements everywhere? For example, I think I recently used `_doc/` somewhere.

If you compare just the ending part, it should be fine (since the language is part of the prefix, the final part remains the same). But when comparing the whole permalink, or the whole file path - then yes, it needs to take care about the possible translation too.

Reply:
> If you compare just the ending part, it should be fine (since the language is part of the prefix, the final part remains the same). But when comparing the whole permalink, or the whole file path - then yes, it needs to take care about the possible translation too.

Ok. In my case, it was:

```
footer.html:{% if page.path contains '_doc/' %}
```

So, not checking the whole path. I've seen that we have `{{ doc_link }}`, but I don't think I've seen one for the doc *path*.